scripts/release-tasks.ts

```diff
@@ -222,7 +222,7 @@ export function runReleaseTasks(opts: BuildOptions, args: ReadonlyArray<string>)
         title: 'Also set "next" npm tag on @stencil/core',
         task: () => {
           const cmd = 'npm';
-          const cmdArgs = ['dist-tag', 'add', '@stencil/core@' + opts.version, 'next'];
+          const cmdArgs = ['dist-tag', 'add', '--otp', '@stencil/core@' + opts.version, 'next'];
```

Comment:
I think the OTP code must be provided on the command line, i.e. `--otp <one-time password>`, per the [docs](https://docs.npmjs.com/cli/v8/commands/npm-dist-tag#description).

Reply:
Ah you're right - that's on me